### PR TITLE
Ol7 updates, introduce new rule, update OVALs and update bash files

### PR DESCRIPTION
**Pull request by Xeicker**
#### Description:

1. Implement network_implement_access_control manual rule
2. Remove STIG id from `install_mcafee_antivirus`
3. Remove oval check from `install_antivirus`
4. Add network_sniffer_disabled bash remediation
5. Add a restart chronyd or ntpd service after remediating chronyd_or_ntpd_set_maxpoll rule
6. Add check of sssd service in OVALs of `sssd_ldap_configure_tls_reqcert` & `sssd_ldap_start_tls`

#### Rationale:

1. This rule covers DISA STIG requirement `OL07-00-040810`
2. The DISA STIG ID doesn't mention a specific antivirus
3. This rule is generic to check for an antivirus, not necessarily Mcafee
4. This rule didn't have a remediation
  - Tested with automatus using existing tests
5. This is to ensure the changes were applied
6. Ensure those rules only fail in case sssd service is enabled

#### Review Hints:

- Check if OVAL removal in `install_antivirus` rule makes sense
- The bash remediation was based in DISA's STIG fix test
- The platform `sssd-ldap` doesn't check for sssd service to be active/enabled.

### CODE REVIEW

**Review by freddieRv — APPROVED** [review]
Probably we don't have to remove the OVAL check for `install_antivirus` if other products actually use it.

Besides that; Changes LGTM.

**Comment by Xeicker** [comment]
> Probably we don't have to remove the OVAL check for `install_antivirus` if other products actually use it.
> 

The logic behind removing it is that this OVAL check is already present in the rule `install_mcafee_antivirus`. So it would be better to use that rule when that OVAL is required

**Comment by jan-cerny** [comment]
@Xeicker please rebase and resolve the conflict

**Comment by freddieRv** [comment]
Failing tests due to container-based test environments:
```
python3.8 tests/automatus.py rule --dontclean --libvirt qemu:///system test-suite-ol7 --datastream build/ssg-ol7-ds.xml network_sniffer_disabled                               
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/ssg_tester/content/logs/rule-custom-2023-02-14-1022/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_network_sniffer_disabled
INFO - Script no_promisc_interfaces.pass.sh using profile (all) OK
INFO - Script promisc_interface_exists.fail.sh using profile (all) OK

python3.8 tests/automatus.py rule --dontclean --libvirt qemu:///system test-suite-ol7 --datastream build/ssg-ol7-ds.xml sssd_ldap_configure_tls_ca_dir                         
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/ssg_tester/content/logs/rule-custom-2023-02-14-1027/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sssd_ldap_configure_tls_ca_dir
INFO - Script domain_not_there.fail.sh using profile xccdf_org.ssgproject.content_profile_stig OK
INFO - Script ldap_tls_cacertdir.pass.sh using profile xccdf_org.ssgproject.content_profile_stig OK
INFO - Script ldap_tls_cacertdir_bad_value.fail.sh using profile xccdf_org.ssgproject.content_profile_stig OK                                                                                                     
INFO - Script ldap_tls_cacertdir_not_absolute_path.fail.sh using profile xccdf_org.ssgproject.content_profile_stig OK                                                                                             
INFO - Script ldap_tls_cacertdir_not_there.fail.sh using profile xccdf_org.ssgproject.content_profile_stig OK 

python3.8 tests/automatus.py rule --dontclean --libvirt qemu:///system test-suite-ol7 --datastream build/ssg-ol7-ds.xml sssd_ldap_configure_tls_reqcert                        
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/ssg_tester/content/logs/rule-custom-2023-02-14-1036/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sssd_ldap_configure_tls_reqcert
WARNING - Script ad_id_provider_and_reqcert_never.notapplicable.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_sssd_ldap_configure_tls_reqcert evaluation resulted in notapplicable                                                                                             
INFO - Script correct_value.pass.sh using profile (all) OK
WARNING - Script id_provider_is_set_to_ad.notapplicable.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_sssd_ldap_configure_tls_reqcert evaluation resulted in notapplicable                                                                                             
INFO - Script ldap_id_provider_and_reqcert_never.fail.sh using profile (all) OK
INFO - Script ldap_tls_reqcert_not_there.fail.sh using profile (all) OK

python3.8 tests/automatus.py rule --dontclean --libvirt qemu:///system test-suite-ol7 --datastream build/ssg-ol7-ds.xml sssd_ldap_start_tls
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/ssg_tester/content/logs/rule-custom-2023-02-14-1041/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_sssd_ldap_start_tls
WARNING - Script ad_id_provider_and_tls_false.notapplicable.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_sssd_ldap_start_tls evaluation resulted in notapplicable                                                                                                         
INFO - Script correct_value.pass.sh using profile (all) OK
WARNING - Script id_provider_is_set_to_ad.notapplicable.sh using profile (all) notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_sssd_ldap_start_tls evaluation resulted in notapplicable                                                                                                         
INFO - Script ldap_id_provider_and_tls_false.fail.sh using profile (all) OK
INFO - Script ldap_use_start_tls_not_there.fail.sh using profile (all) OK
```

**Comment by freddieRv** [comment]
Since most commits are authored by me, could someone else kindly take a look?
@marcusburghardt @vojtapolasek

**Review by freddieRv — APPROVED** [review]

File: tests/shared/sssd.conf [hunk 1]
```diff
@@ -1,35 +1,19 @@
 [domain/default]
```
Comment (vojtapolasek) [inline tests/shared/sssd.conf:1]: What is the reason for changes in this file? It is just a sample file.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello,
thank you for the PR. I have some feedback for you.
Most of the feedback is contained in individual comments.
I have one general remark; please consider introducing changes in individual pull requests. This PR touches severa rules which are not dependent on each other and it might therefore make the review process longer as different rules might need different amount of modifications.
Please consider reverting the change which removes OVAL from install_antivirus. I understand your reasoning, but This rule is used in several profiles and I think this change deserves a different approach. We can discuss it in a separate pull request.

**Comment by jan-cerny** [comment]
Per CONTRIBUTING.md, I'm closing it because of inactivity for more than 30 days. Feel free to reopen it if you still plan to address the feedback.

**Pull request closed without merging**